doc/source/whatsnew/v0.23.0.txt

```diff
@@ -119,6 +119,56 @@ Current Behavior
 
     s.rank(na_option='top')
 
+
+Extending Pandas Objects with New Accessors
```

Comment:
Thoughts on removing this section in the whatsnew? This may be too niche for most users to care about.